article/index.md

```diff
@@ -2,19 +2,22 @@
 
 ![Profile Picture](./ProfielFotoPriveRound.png) Ringo De Smet
 
-**Guest Article:** [Ringo De Smet](https://www.linkedin.com/in/ringodesmet/), Founder of [Cumundi](https://www.cumundi.cloud), standardizes on Pulumi for writing infrastructure as code libraries to be delivered as reusable building blocks. Pulumi enables them to rapidly iterate through the build-test-release cycle of these building blocks.
+**Guest Article:** [Ringo De Smet](https://www.linkedin.com/in/ringodesmet/), Founder of [Cumundi](https://www.cumundi.cloud), standardizes on Pulumi for writing infrastructure as code libraries to be delivered as reusable building blocks. Pulumi enables him to rapidly iterate through the build-test-release cycle of these building blocks.
 
-Cumundi helps companies adopt cloud in a more integral way. Most companies are developping the same 75% of underlying infrastructure, but mostly with limited investment in integrating all the non-functionals of running applications in the cloud due to a shortage of people and time.
+Cumundi helps companies adopt cloud in a more integral way. Companies are developing the same 75% of underlying infrastructure. However, due to a shortage of people and time, there is a limited investment in integrating all the non-functionals of running applications in the cloud.
 
 ## How Pulumi code keeps up with changing requirements
 
-Cumundi builds reusable libraries for our customers to set up their infrastructure with a lot of best practices integrated. These best practices span the range of non-functionals which are most of the time not taken up with the same focus as application feature development. Where these non-functionals can be integrated in infrastructure setups, we make sure these are taken care of in our code libraries. We focus on all three big cloud providers and try to provide application focused infrastructure libraries.
+@Ringo: ofwel over Cumundi (en dus they en their...) ofwel we, our, ... Ik denk dat je meer our/I gebruikt, dus... (maar ik heb niet overal aangepast).
```

Comment:
@majadhondt the first paragraph is indeed written in another style from the rest. If you look at [this other guest blog post](https://www.pulumi.com/blog/bigdata-boutique-guest-post/), the first paragraph is like it is written by Pulumi, while the rest is written by me/us. With this in mind, is it OK to leave the differences with `they` & `we`?